pypeman/tests/test_nodes.py

```diff
@@ -497,6 +510,26 @@ def test_httprequest_node(self):
             mock_session.request.assert_called_once_with(**req_kwargs3)
             mock_load_cert_chain.assert_called_once_with(client_cert[0], client_cert[1])
 
+    def test_httpjsonrequest_node(self):
+        """ Whether HttpJsonRequest node is functional """
+        channel = FakeChannel(self.loop)
+        url = 'http://url/titi/tata'
+        http_jsonnode = nodes.HttpJsonRequest(url=url, verify=False)
+        http_jsonnode.channel = channel
+        data = {"titi": "tata"}
+        jsondata = json.dumps(data)
+        emptymsg = message.Message()
+        with mock.patch('pypeman.contrib.http.HttpRequest.handle_request') as mock_http_get:
+            if sys.version_info < (3, 8):
+                # TODO rm when pypeman drop 3.7 support
```

Comment:
drop -> drops

Reply:
now that we think about it:
rm in MR, that remoces 3.7

Reply:
done

Reply:
And yes I agree